src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -215,6 +216,23 @@ public function validated($key = null, $default = null)
         return data_get($this->validator->validated(), $key, $default);
     }
 
+    /**
+     * Get the validated data from the request except for a specified array of items.
+     *
+     * @param  array|mixed  $keys
```

Comment:
`mixed` can never be combined with something else.

Reply:
Thank you @driesvints, fixed